core-bundle/src/Migration/Version409/CeAccessMigration.php

```diff
@@ -77,6 +77,6 @@ public function run(): MigrationResult
             ':fields' => serialize(array_keys($GLOBALS['TL_FFL'])),
         ]);
 
-        return new MigrationResult(true, '');
+        return $this->createResult(true);
```

Comment:
This should not be part of this PR, should it?

Reply:
that's because I upstream-merged 4.9 into master. It will be gone when rebasing once #1776 is merged.